elytron/src/main/resources/schema/wildfly-elytron_4_0.xsd

```diff
@@ -2245,6 +2245,7 @@
             <xs:element name="constant-role-mapper" type="constantRoleMapperType" minOccurs="0" maxOccurs="unbounded" />
             <xs:element name="custom-role-mapper" type="customRoleMapperType" minOccurs="0" maxOccurs="unbounded" />
             <xs:element name="logical-role-mapper" type="logicalRoleMapperType" minOccurs="0" maxOccurs="unbounded" />
+            <xs:element name="mapped-role-mapper" type="mappedRoleMapperType" minOccurs="0" maxOccurs="unbounded" />
```

Comment:
Please, when adding new elements, add example of their usage into `elytron-subsystem-4.0.xml`